features/admin.logs.v1/pages/logs-page.tsx

```diff
@@ -72,80 +70,92 @@ const LogsPage: FunctionComponent<LogsPageInterface> = (
+                    menuItem: (
+                        <MenuItem key="text" className="item-with-chip">
+                            { t("extensions:develop.monitor.logs.tabs.diagnostic") }
+                        </MenuItem>
+                    ),
+                    render: renderLogContentDiagnosticNew
+                });
+        }
+
+        {
+            featureConfig.auditLogs?.enabled &&
+                hasAuditLogAccessPermissions &&
+                panes.push({
+                    componentId: "audit-logs",
+                    menuItem: (
+                        <MenuItem key="text" className="item-with-chip">
+                            { t("extensions:develop.monitor.logs.tabs.audit") }
+                            <Chip size="small" label={ t("common:beta") } className="oxygen-chip-beta" />
+                        </MenuItem>
+                    ),
+                    render: renderLogContentAuditNew
+                });
         }
 
         return panes;
     };
 
-    const renderLogContentDiagnosticNew = () : ReactElement => {
-        return (
-            <DiagnosticLogsPage
-                data-componentid={ componentId }
-            />
-        );
+    const renderLogContentDiagnosticNew = (): ReactElement => {
+        return <DiagnosticLogsPage data-componentid={ componentId } />;
     };
 
-    const renderLogContentAuditNew = () : ReactElement => {
-        return (
-            <AuditLogsPage/>
-        );
+    const renderLogContentAuditNew = (): ReactElement => {
+        return <AuditLogsPage />;
+    };
+
+    const handleSettingsButton = () => {
+        history.push(AppConstants.getPaths().get("LOGS_SETTINGS"));
     };
 
     return (
         <div className="diagnostic-logs">
             <PageLayout
                 title={ t("extensions:develop.monitor.pageHeader.title") }
                 pageTitle="Logs"
-                description={ (
-                    <>
+                description={
+                    (<>
                         { t("extensions:develop.monitor.pageHeader.description") }
-                        <DocumentationLink
-                            link={ getLink("manage.logs.learnMore") }
-                        >
+                        <DocumentationLink link={ getLink("manage.logs.learnMore") }>
                             { t("common:learnMore") }
                         </DocumentationLink>
-                    </>
-                ) }
+                    </>)
+                }
+                action={
+                    (<Popup
+                        trigger={
+                            (<Button
+                                data-componentid={ "applications-settings-button" }
+                                icon={ GearIcon }
+                                onClick={ handleSettingsButton }
+                            />)
+                        }
+                        content={ t("applications:forms.applicationsSettings.title") }
```

Comment:
Is it expected to get the popup title from `applications` namespace.
```suggestion
                        content={ t("applications:forms.applicationsSettings.title") }
```